### DISCUSSION TITLE
Way to increase max size for the scene?

### DISCUSSION THREAD

**Discussion by raspi0124** [mozilla/hubs#3152] [Spoke Questions] [answered]
Hi, I deployed Hubs Cloud on AWS and currently working on making the scene on Spoke. I'm getting Scene is too large error on my custom Hubs server. Is there any other solution then reducing models? Like editing code to increase the 128MB limit?

**Comment by robinkwilson** [comment, marked as answer]
Unfortunately changing the 128MB limit is difficult because the spoke scene limit is bottle necked in spoke and reticulum (internal server). If there's a way to optimize the scene / bundled textures, that is definitely the preferred solution. Or you can place the assets inside the room afterward.

To place assets in room afterward: 
For example if you have a large video or for any other assets, make the scene without the video in Spoke, then open the scene in a room, then add the video and pin it.

**Reply by nikimamusic** [reply]
Hi @robinkwilson, there is a limit to the size assets using this method (place assets in room afterward)? Instead of videos or images can I place 3d assets (specially, animated ones! like characters) in room afterward?

Thanks!

**Reply by aelainelong** [reply]
@robinkwilson Just following up on @nikimamusic's question above -- does placing 3D or video assets into a Hubs room afterward and pinning let us get around the recommended <50k poly/128mb size limitations?
